### ISSUE TITLE
ESLint complains about duplicate plugin import, when importing using in parent config file

### ISSUE THREAD

**Issue by lukasgjetting** [eslint/eslint#13436]
## Thanks

First of all, I would like to take a moment to just give a big **thanks** to everyone who has contributed to ESLint.

You of course already know this, but your work is **very much** appreciated.

## Environment

* **ESLint Version:** v7.3.1
* **Node Version:** v12.18.1
* **npm Version:** v6.14.5

*Note:  
`eslint --env-info` gives `Not found` for both `Local ESLint version` and `Global ESLint version`. Not sure whether this is relevant?*

### `package.json` files

<details>
<summary>package.json</summary>

```js
{
  ...
  "scripts": {
    "lint-api": "cd api && eslint . --ext .js,.jsx --max-warnings=0",
  },
  "dependencies": {
    "concurrently": "^5.1.0",
    "dotenv": "^8.2.0",
    "dotenv-cli": "^3.1.0",
    "eslint-config-airbnb": "^18.0.1",
    "eslint-config-react-app": "^5.2.0",
    "eslint-plugin-css-modules": "^2.11.0",
    "express-jwt": "^5.3.1",
    "jwks-rsa": "^1.7.0"
  },
  "devDependencies": {
    "@typescript-eslint/parser": "^2.33.0",
    "cypress": "^3.8.3",
    "cypress-react-selector": "^0.1.8",
    "eslint": "^7.3.1",
    "eslint-plugin-cypress": "^2.10.3",
    "eslint-plugin-flowtype": "^4.6.0",
    "eslint-plugin-jsx-a11y": "^6.2.3",
    "eslint-plugin-modules-newline": "^0.0.4",
    "eslint-plugin-react": "^7.14.3",
    "eslint-plugin-react-hooks": "^1.7.0",
    "eslint_d": "^9.1.0",
    "husky": "^4.2.5",
    "typescript": "^3.9.2"
  },
  ...
}
```
</details>
<details>
<summary>api/package.json</summary>

```js
{
  ...
  "dependencies": {
    "aws-sdk": "^2.635.0",
    "bluebird": "^3.7.2",
    "cors": "^2.8.5",
    "dotenv": "^8.2.0",
    "express": "^4.17.1",
    "express-fileupload": "^1.1.6",
    "express-jwt": "^5.3.1",
    "express-query-boolean": "^2.0.0",
    "handlebars": "^4.7.6",
    "handlebars-layouts": "^3.1.4",
    "imap-simple": "^4.3.0",
    "jwks-rsa": "^1.7.0",
    "mailparser": "^2.7.7",
    "md5": "^2.2.1",
    "moment": "^2.24.0",
    "morgan": "^1.9.1",
    "mustache": "^4.0.1",
    "node-cron": "^2.0.3",
    "node-fetch": "^2.6.0",
    "node-html-parser": "^1.2.16",
    "pdf-puppeteer": "^1.1.10",
    "pg": "^7.18.2",
    "pg-hstore": "^2.3.3",
    "sequelize": "^5.21.5",
    "short-uuid": "^3.1.1"
  },
  "devDependencies": {
    "@babel/cli": "^7.8.4",
    "@babel/core": "^7.8.4",
    "@babel/node": "^7.8.4",
    "@babel/preset-env": "^7.8.4",
    "eslint": "^7.3.1",
    "babel-plugin-module-resolver": "^4.0.0",
    "eslint-import-resolver-babel-module": "^5.1.2",
    "eslint-plugin-import": "^2.20.2",
    "nodemon": "^2.0.2",
    "sequelize-cli": "^5.5.1",
    "jest": "^26.0.1"
  }
  ...
}
```
</details>

## Parser

Default

## Configuration

<details>
<summary>Root configuration</summary>

```js
const config = {
	extends: [
		'airbnb',
	],
	plugins: [
		'modules-newline',
	],
};

module.exports = config;
```

</details>

<details>
<summary>Child configuration</summary>

```js
{
	"extends": ["../.eslintrc"],
	"settings": {
		"import/resolver": {
			"babel-module": {}
		}
	},
	"env": {
		"jest": true
	}
}
```
</details>

## Reproduction
```bash
yarn lint-api
```

**What did you expect to happen?**

ESLint runs without any issues.

**What actually happened? Please include the actual, raw output from ESLint.**

ESLint throws an error regarding duplicate `import` plugin.

<details>
<summary>Output</summary>

```
$ yarn lint-api
yarn run v1.21.1
$ cd api && eslint . --ext .js,.jsx --max-warnings=0

Oops! Something went wrong! :(

ESLint: 7.3.1

ESLint couldn't determine the plugin "import" uniquely.

- C:\Users\lukas\work\api\node_modules\eslint-plugin-import\lib\index.js (loaded in ".eslintrc » ../.eslintrc » eslint-config-airbnb » C:\Users\lukas\work\node_modules\eslint-config-airbnb-base\index.js » C:\Users\lukas\work\node_modules\eslint-config-airbnb-base\rules\imports.js")
-  (loaded in "..\.eslintrc.js » eslint-config-airbnb » C:\Users\lukas\work\node_modules\eslint-config-airbnb-base\index.js » C:\Users\lukas\work\node_modules\eslint-config-airbnb-base\rules\imports.js")

Please remove the "plugins" setting from either config or remove either plugin installation.

If you still can't figure out the problem, please stop by https://eslint.org/chat to chat with the team.

error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```
</details>

**Are you willing to submit a pull request to fix this bug?**

Yes.

However, I would have no idea where to start.

I would need a bit of guidance and a nudge in the right direction.

**Comment by lukasgjetting** [comment]
# Oh boy.

I was certain I had already exhausted all possible options trying to solve it. I hadn't.

# Solution

Adding `root: true` to the root `.eslintrc` file seems to solve the issue.

Source: https://github.com/eslint/eslint/issues/13385#issuecomment-641252879
